azurerm/internal/services/network/tests/resource_arm_public_ip_test.go

```diff
@@ -112,22 +112,6 @@ func TestAccAzureRMPublicIpStatic_basic_withDNSLabel(t *testing.T) {
 	})
 }
 
-func TestAccAzureRMPublicIpStatic_standard_withIPv6_fails(t *testing.T) {
```

Comment:
Could we invert this test so that it is checks that it succeeds?

Reply:
@katbyte done, PTAL!